infobserve/common/config.py

```diff
@@ -78,5 +76,10 @@ async def init_db(self):
                 except asyncpg.exceptions.DuplicateTableError:
                     print("Duplicate Table Error Raised the sql init script failed.")
 
+    async def init_redis_pool(self):
```

Comment:
Maybe we shouldn't initialize stuff in the `config.py` file. Maybe add an `initialize.py` (🤮) or rename this from `config.py` to something else, and actually do *all* initializations here.

Reply:
I changed it all i moved all the initialization logic to the pools module with the thought of implementing database managers in the future if we need them. But yeah the pools create and provide the connections and should contain the methods that setup the databases for now.